index.html

```diff
@@ -1,389 +1,412 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-<meta charset="utf-8">
-<meta name="viewport" content="width=device-width, initial-scale=1">
-<title>Jalaz Kumar - Full Stack Developer & Aspiring Data Scientist</title>
-<meta name="description" content="">
-<meta name="author" content="">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Jalaz Kumar - Full Stack Developer & Aspiring Data Scientist</title>
+    <meta name="description" content="">
+    <meta name="author" content="">

```

Comment:
Why have you indented each & everything up? Kindly unindent each & every line of code & commit it. So, that the real contribution which you made for the **Project Section Design** is clearly visible.